base/util.jl

```diff
@@ -467,36 +479,30 @@ function getpass(prompt::AbstractString)
 end
 else
 function getpass(prompt::AbstractString)
```

Comment:
Should `prompt` have a default value, e.g. `"Password"`?

Reply:
It can always be added later if someone else feels strongly about it.